app/src/main/java/com/mensinator/app/CalendarScreen.kt

```diff
@@ -90,6 +91,10 @@ fun CalendarScreen() {
     var previousFirstPeriodDate by remember { mutableStateOf<LocalDate?>(null) }
     val colorMap = ColorSource.getColorMap(isDarkMode())
 
+    val initPeriodMessage = dbHelper.getSettingByKey("period_notification_message")?.value.toString()
```

Comment:
Same problematic code (now I noticed it's even duplicated). Arguably this could even be included in the DbHelper (you can call `context.getString()` from within `PeriodDatabaseHelper` or it should be another class. Not sure.

Reply:
Yes unfortunately it wasn't neatly handled with an ?: since I couldn't initialise the value as null in the DB, will try and look at a solution

Reply:
This should be covered by the `getStringSettingByKey` func now